hms-app/src/components/buttons/CardButton.tsx

```diff
@@ -122,6 +124,13 @@ export default function CardButtons(props: IProps) {
     </Modal>
   )
 
+  const uploadButton = () => {
+    return (
+      type === IdeaCardType.AllIdeas &&
```

Comment:
do we really need that check? 
I would say we would need to check if its the admin page or not
on all pages it should only be visible when hackathon started, excpect for the admin page there it should be always visible

Reply:
that does makes sense. I forgot about the admin page